### DISCUSSION TITLE
don`t work events

### DISCUSSION THREAD

**Discussion by crazych1** [blakeblackshear/frigate#15639] [Ask A Question] [answered]
I'm trying to use a construction where 7 days of continuous recording and 30 days of motion recordings are stored, but for some reason it gives an error on the events directive.
`record:
  enabled: True
  retain:
    days: 3
    mode: all
  events:
    retain:
      default: 30
      mode: motion`

![2024-12-22_211235](https://github.com/user-attachments/assets/d22786d8-6033-454c-b40f-d55c344f0193)

how to solve the problem?

**Comment by hawkeye217** [comment]
What version of Frigate are you using?

**Comment by crazych1** [comment]
ghcr.io/blakeblackshear/frigate:0.15.0-beta3

**Reply by hawkeye217** [reply, marked as answer]
You should read the release notes, the `record` config has changed in 0.15.

https://github.com/blakeblackshear/frigate/releases/tag/v0.15.0-beta3

**Comment by crazych1** [comment]
Thank you very much, you helped me a lot to understand the problems.

**Reply by hawkeye217** [reply]
Happy to help.
